### DISCUSSION TITLE
현재 구현해야하는 공격 스킬들 목록

### DISCUSSION THREAD

**Discussion by YOON-JAEYEONG** [YOON-JAEYEONG/Student_Survivor#21] [General]
~~마이크로프로세서~~ 윤재영
시스템프로그래밍및보안
~~데이터베이스~~ 하성헌
~~IoT~~ 김시웅
~~클라우드컴퓨팅~~ 김시웅

구현 중 혹은 구현 완료
~~논리회로~~ 이성동
~~자료구조~~ 윤재영
~~데이터분석과머신러닝~~ 김시웅
~~웹 프로그래밍~~ 윤재영
~~컴퓨터그래픽스~~ 김시웅

**Comment by YOON-JAEYEONG** [comment]
제가 일단 웹프랑 자구 맡겠습니다

**Reply by YOON-JAEYEONG** [reply]
마프 하겠습니다

**Comment by rlatldnd1** [comment]
나는 컴그

**Reply by rlatldnd1** [reply]
머신러닝 할게요

**Reply by rlatldnd1** [reply]
클라우드 할게요

**Reply by rlatldnd1** [reply]
iot 할게욜

**Comment by gnodss** [comment]
저는 논리회로 해볼게요
